<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HostRoomUi</class>
 <widget class="QMainWindow" name="HostRoomUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="le_chatsend">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>520</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>520</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_view">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>701</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">弹

幕

显

示

区

域</string>
    </property>
    <property name="text">
     <string>弹

幕

显

示

区

域</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_barrage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>691</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>50</y>
      <width>131</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>50</y>
      <width>141</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>696</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>房间名</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_RoomName"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>金钱</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="le_RoomMoney"/>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>积分</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLineEdit" name="le_RoomLevel"/>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>主播名</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLineEdit" name="le_HostName"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>575</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>当前账户</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_account">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>账户余额</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="le_money">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>账户积分</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLineEdit" name="le_score">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>451</y>
      <width>571</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>    礼    物</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>鱼丸</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>鱼翅</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>跑车</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>飞机</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>火箭</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>聊天记录</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>聊天刷新</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>10</y>
      <width>143</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>群成员列表</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string>成员</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>lab_barrage</zorder>
   <zorder>label</zorder>
   <zorder>le_account</zorder>
   <zorder>label_2</zorder>
   <zorder>le_money</zorder>
   <zorder>label_3</zorder>
   <zorder>le_score</zorder>
   <zorder>le_chatsend</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_4</zorder>
   <zorder>le_RoomName</zorder>
   <zorder>le_RoomMoney</zorder>
   <zorder>le_RoomLevel</zorder>
   <zorder>le_HostName</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>lab_view</zorder>
   <zorder>textEdit</zorder>
   <zorder>listWidget</zorder>
   <zorder>label_9</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>label_10</zorder>
   <zorder>pushButton_8</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1107</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
